docpages/example_programs/the_basics/using_callback_functions.md

```diff
@@ -0,0 +1,97 @@
+\page callback-functions Using Callback Functions
+
+When you create or get an object from Discord, you send the request to its API and in return you get either an error or the object you requested/created. Let's create a message and get it:
+
+~~~~~~~~~~~~~~{.cpp}
+#include <dpp/dpp.h>
+#include <fmt/format.h> // used to format text, install and define FMT_HEADER_ONLY to use
```

Comment:
I would suggest not depending on fmt for an example, we want examples to be super simple and easy to copy, paste, and run

Reply:
+1

Reply:
Mission SimplifyThat activated